homu/server.py

```diff
@@ -29,8 +29,20 @@ def index():
 
 @get('/queue/<repo_label>')
 def queue(repo_label):
-    repo = g.repos[repo_label]
-    pull_states = sorted(g.states[repo_label].values())
+    if repo_label == "all":
+        labels = list(g.repos.keys())
+    else:
+        labels = repo_label.split("+")
+
+    states = []
+    for label in labels:
+        repo = g.repos[label]
+        new_states = g.states[label].values()
+        for state in new_states:
+            state.repo = repo
```

Comment:
I'm not sure why this code is needed. Isn't `state.repo` already set by the constructor of `PullReqState`?